packages/lit/src/components/app/app.ts

```diff
@@ -41,6 +41,18 @@ export class App extends LitElement {
     defineCustomElements();
     this.addEventListener(EVENT_NAMES.CONFIG_CHANGE, this.#handleConfigUpdate);
     this.addEventListener(EVENT_NAMES.TEMPLATE_CHANGE, this.#handleTemplateChange);
+
+    // Parse template selection from query param: ?templates=/group/page (dynamic)
+    try {
+      const params = new URL(globalThis.location.href).searchParams.get('templates');
+      if (params) {
+        const normalized = params.startsWith('/') ? params : `/${params}`;
+        const [, group, page] = normalized.split('/');
+        if (group && page) this.selectedTemplatePath = `/${group}/${page}`;
```

Comment:
Niet heel belangrijk maar je doet al een operatie die je kan gebruiken voor je normalisatie.

```suggestion
      const params = new URL(globalThis.location.href).searchParams.get('templates');
      if (params) {
        const [group, page] = params.split('/').filter(Boolean); // Skip empty path components
        if (group && page) this.selectedTemplatePath = `/${group}/${page}`;
      }
```